do_files_push.go

```diff
@@ -241,32 +245,58 @@ func doFilesPush(
 		response, err := client.UploadFile(project, request)
 
 		if err != nil {
-			return NewError(
-				hierr.Errorf(
-					err,
-					`unable to upload file "%s"`,
-					file,
-				),
+			if returnError(err) {
+				return NewError(
+					hierr.Errorf(
+						err,
+						`unable to upload file "%s"`,
+						file,
+					),
+
+					`Check, that you have enough permissions to upload file to`+
+						` the specified project`,
+				)
+			}
+			_, _ = fmt.Fprintln(os.Stderr, "Unable to upload file "+file)
+			failedFiles = append(failedFiles, file)
+		} else {
+			status := "new"
+			if response.Overwritten {
+				status = "overwritten"
+			}
 
-				`Check, that you have enough permissions to upload file to`+
-					` the specified project`,
+			fmt.Printf(
+				"%s (%s) %s [%d strings %d words]\n",
+				uri,
+				request.FileType,
+				status,
+				response.StringCount,
+				response.WordCount,
 			)
 		}
+	}
 
-		status := "new"
-		if response.Overwritten {
-			status = "overwritten"
-		}
+	if len(failedFiles) != 0 {
+		result = NewError(fmt.Errorf("failed to upload %d files", len(failedFiles)), "failed to upload files "+strings.Join(failedFiles, ", "))
+	}
 
-		fmt.Printf(
-			"%s (%s) %s [%d strings %d words]\n",
-			uri,
-			request.FileType,
-			status,
-			response.StringCount,
-			response.WordCount,
-		)
+	return result
+}
+
+func returnError(err error) bool {
+	if err.Error() == "failed to upload original file: unable to authenticate: "+
```

Comment:
I'm lazy to check what do we put into `err` in SDK, but i think it should contain ALL information from error response:
* HTTP status
* smartling error `code`
* array of errors

A couple of error examples
```json
{
  "response": {
    "code": "AUTHENTICATION_ERROR",
    "errors": [
      {
        "key": "invalid_token",
        "message": "Token is missing.",
        "details": null
      }
    ]
  }
}

{
    "response": {
        "code": "GENERAL_ERROR",
        "errors": [
            {
                "key": "parse.error",
                "message": "There was a problem loading your file. java.lang.IllegalArgumentException: File already belongs to namespace 'iap-instructions.xml'. Namespace cannot be changed to 'accessibility'.",
                "details": {
                    "errorId": "siqdksfd"
                }
            }
        ]
    }
}
```

So ideally you should compare fields: status code, `code` or `key`